openedx/core/djangoapps/schedules/models.py

```diff
@@ -54,14 +54,14 @@ class ScheduleConfig(ConfigurationModel):
     KEY_FIELDS = ('site',)
 
     site = models.ForeignKey(Site, on_delete=models.CASCADE)
-    create_schedules = models.BooleanField(default=False)
+    create_schedules = models.BooleanField(default=False)  # deprecated, do not use
```

Comment:
Maybe worth adding the deprecatedness as an actual help-string on the field?

Reply:
I believe help string changes cause a migration. I'm hoping to drop these columns very shortly anyway in my nudge 2.0 pr.